kodi-cli

```diff
@@ -21,7 +21,7 @@ XBMC_PASS="${XBMC_PASS:-}"
 LOCK=false
 
 function xbmc_req {
- curl -s -i -X POST --header "Content-Type: application/json" -d "$1" http://$XBMC_USER:$XBMC_PASS@$XBMC_HOST:$XBMC_PORT/jsonrpc >/dev/null
+ curl -v -s -i -X POST --header "Content-Type: application/json" -d "$1" http://$XBMC_USER:$XBMC_PASS@$XBMC_HOST:$XBMC_PORT/jsonrpc >/dev/null
 }
```

Comment:
Do you need to add the -v tag here, or was that just during debugging? I don't have/use xbmctorrent so I cannot test.